### ISSUE TITLE
Dajax's _clean() breaks <textarea> tags

### ISSUE THREAD

**Issue by tehfink** [jorgebastida/django-dajax#21]
line 85 of Dajax.py is the problem:

```
return unicode(data).replace('\n','').replace('\r','')
```

example ajax.py:

```
def edit_contact(request, object):
    dajax = Dajax()
    …
    html = render_to_string('contact_form.html', {'formset': formset})
    dajax.assign('#contactform', 'innerHTML', html)
    return dajax.json()
```

a `textarea` from the string `html` looks like:

```
<textarea id='street_address'>
    line1\nline2\nline3
</textarea>
```

but dajax returns this textarea:

```
<textarea id='street_address'>
    line1line2line3
</textarea>
```

**Comment by rolnxyz** [comment]
Yea, I have the same problem with this bug.

I ended up creating another function assign_dirty that doesn't clean.

**Comment by jorgebastida** [comment]
Fixed on 'develop', ready for 0.9.2

Thanks!

**Comment by jorgebastida** [comment]
0.9.2 Released http://pypi.python.org/pypi/django-dajax/0.9.2
